deploy/addons/gvisor/gvisor-containerd-shim.toml

```diff
@@ -0,0 +1,5 @@
+runc_shim = "/bin/containerd-shim"
+[runsc_config]
+ debug-log="/tmp/runsc/"
+ debug="true"
+ log="/tmp/runsc/out.log"
```

Comment:
The --log option is set by gvisor-containerd-shim to be a path per sandbox container created so this option shouldn't be necessary.

Reply:
See: https://github.com/google/gvisor-containerd-shim/blob/master/pkg/go-runsc/runsc.go#L443